support-frontend/assets/helpers/forms/recaptcha.ts

```diff
@@ -1,14 +1,12 @@
-const loadRecaptchaV2 = () =>
+const loadRecaptchaV2 = (): Promise<void> =>
 	new Promise<void>((resolve, reject) => {
 		const recaptchaScript = document.createElement('script');
 		recaptchaScript.src =
 			'https://www.google.com/recaptcha/api.js?onload=v2OnloadCallback&render=explicit';
 		recaptchaScript.onerror = reject;
-		recaptchaScript.onload = resolve;
+		recaptchaScript.onload = () => resolve;
```

Comment:
Should this be script.onload = () => resolve();, as we want to call resolve on onload?